syncd/syncd_flex_counter.cpp

```diff
@@ -522,9 +543,36 @@ void FlexCounter::flexCounterThread(void)
     while (m_runFlexCounterThread)
     {
         {
-            std::lock_guard<std::mutex> lock(g_mutex);
-            collectCounters(countersTable);
-            runPlugins(db);
+            std::map<std::string, std::vector<swss::FieldValueTuple>> portStats;
+            std::map<std::string, std::vector<swss::FieldValueTuple>> queueStats;
+            std::map<std::string, std::vector<swss::FieldValueTuple>> queueAttrs;
+            collectCounters(portStats, queueStats, queueAttrs);
+            if (m_pollInterval == PFC_WD_POLL_MSECS)
```

Comment:
we should probably be able to change the polling internal, use the polling interval to determine whether we should runPlugins or not is not very readable.